tests/Unit/Tracking/MembershipTrackingTest.php

```diff
@@ -5,12 +5,12 @@
 use PHPUnit\Framework\Attributes\CoversClass;
```

Comment:
The classes are missing for the following four lines, scrutinizer is complaining about it.
I guess because this is just a warning, we will solve it later.